ecs-cli/modules/cli/regcreds/regcreds_app.go

```diff
@@ -306,3 +326,26 @@ func validateOutputOptions(outputDir string, skipOutput bool) error {
 	}
 	return nil
 }
+
+func tagRegistryCredentials(creds map[string]regcredio.CredsOutputEntry, tags map[string]*string, taggingClient tagging.Client) error {
+	var arns []*string
+
+	for _, credInfo := range creds {
+		arns = append(arns, aws.String(credInfo.CredentialARN))
+	}
+
+	input := &taggngSDK.TagResourcesInput{
+		ResourceARNList: arns,
+		Tags:            tags,
+	}
+	output, err := taggingClient.TagResources(input)
+	if err != nil {
+		return err
+	}
+
+	for resource, info := range output.FailedResourcesMap {
+		return fmt.Errorf("Failed to tag resource %s; error=%s", resource, *info.ErrorMessage)
```

Comment:
can we add test for this error case (FailedResourceMap != nil) and the API error return (line 342)?